src/main/java/com/basiscomponents/db/util/DataRowJsonMapper.java

```diff
@@ -70,16 +100,45 @@ public static DataRow fromJson(final String in, final DataRow meta, JsonElement
 			} else {
 				handleOldFormat(navigation, dr);
 			}
-			if (attributes!=null){
-				attributes.getAsJsonObject().entrySet().stream().forEach(entry->dr.setAttribute(entry.getKey(),entry.getValue().getAsString()));
+			if (root.get(0).has(ResultSetJsonMapper.ATTRIBUTES)) {
```

Comment:
Since in my implementation, I'm not getting an `attributes` param of type `JsonElement`, I had to reimplement this portion using Jackson library methods.

I can try to implement a similar looking lambda instead of Iterator approach with Jackson lib too. Would you rather prefer that?

Reply:
I think it looks cleaner. Also I prefer not to use Iterators if you don't need them. WE already have too much of them in our code